lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -87,3 +87,13 @@ def test_multipage_keep_empty():
             pdf.savefig()
         assert os.path.exists(filename)
     os.remove(filename)
+
+
+@image_comparison(baseline_images=['test_type1font_encoding'],
+                  extensions=['pdf'])
+def test_type1font_encoding():
+    # This doesn't work
```

Comment:
That is a typo from modifying the bug report example to be a test, should have been removed.